### PR TITLE
Adapt docs to new preferred behaviour in 'conan get' command

### PR DESCRIPTION
**Pull request by jgsogo**
Docs for https://github.com/conan-io/conan/pull/4494

### CODE REVIEW

File: reference/commands/consumer/get.rst [hunk 1]
```diff
@@ -58,7 +56,7 @@ Gets a file or list a directory of a given reference or package.
 
   .. code-block:: bash
 
-      $ conan get zlib/1.2.11@conan/stable -p 09512ff863f37e98ed748eadd9c6df3e4ea424a8
+      $ conan get zlib/1.2.11@conan/stable:09512ff863f37e98ed748eadd9c6df3e4ea424a8
```
Comment (uilianries) [inline reference/commands/consumer/get.rst]: This is more readable :smile_cat: 

But, as `-p` is suppressed now, but still supported, so why not add a note about that? If I read this documentation, I'll understand it like a breaking change, where `-p` is no longer supported.

Comment (danimtb) [reply reference/commands/consumer/get.rst]: It has been removed from the help command but the parameter is still usable. Maybe we should leave a deprecation notice with the example below. WDYT?

Comment (uilianries) [reply reference/commands/consumer/get.rst]: A deprecation notice sounds good!

Comment (uilianries) [reply reference/commands/consumer/get.rst]: a deprecation notice sounds good!

**Comment by jgsogo** [comment]
This is the output now:

```
⇒  conan get --help
usage: conan get [-h] [-p PACKAGE] [-r REMOTE] [-raw] reference [path]

Gets a file or list a directory of a given reference or package.

positional arguments:
  reference             package recipe reference
  path                  Path to the file or directory. If not specified will
                        get the conanfile if only a reference is specified and
                        a conaninfo.txt file contents if the package is also
                        specified

optional arguments:
  -h, --help            show this help message and exit
  -p PACKAGE, --package PACKAGE
                        Package ID (deprecated, use full reference)
  -r REMOTE, --remote REMOTE
                        Get from this specific remote
  -raw, --raw           Do not decorate the text
```

Shall I use a `[DEPRECATED]` string, @danimtb?

**Comment by danimtb** [comment]
I really like ``[ ]`` for "tagging" things as deprecated, obsolete, experimental and stuff like that. But not mandatory though...

**Comment by jgsogo** [comment]
I like it too, but if I use the _label_ `[DEPRECATED]`, where do I put the reason or alternative? Something like `[DEPRECATED: use full reference instead]`?

**Comment by danimtb** [comment]
yeah, it could be fine like that!

**Pull request merged**